pkg/service/service.go

```diff
@@ -0,0 +1,117 @@
+package service
+
+import (
+	"github.com/readium/go-toolkit/pkg/fetcher"
+	"github.com/readium/go-toolkit/pkg/manifest"
+)
+
+// Base interface to be implemented by all publication services.
+type Service interface {
```

Comment:
Since services are core to the `Publication` object, we can keep these under the `pub` package. I think it makes for a clearer type too: `pub.PositionsService` vs `service.Positions`.